### PR TITLE
Adding timestamps to the items in the lists as well as to the "Have i…

### PR DESCRIPTION
**Pull request by Pyrodius0**
…t" card on the item page.

### CODE REVIEW

File: resources/views/character/partials/header.blade.php [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+    @if (isset($character->pivot->created_at))
+    <li>
+        <ul class="list-inline">
+            <li class="list-inline-item text-muted small">
+                  Recieved: <span class="js-watchable-timestamp js-timestamp-title" data-timestamp="{{ $character->pivot->created_at }}"></span> ago
+            </li>
+        </ul>
+    </li>
+    @endif
+    <li/>
```
Comment (Lemmings19) [inline resources/views/character/partials/header.blade.php:11]: I think this should be removed from the PR. This template is for displaying character names on pages generated by PHP, and I can't see a use case where we'd want this to appear above the character's name.

Comment (Pyrodius0) [reply resources/views/character/partials/header.blade.php:11]: I do not think this feature should be removed. This provides the time the item was received on the Loot Item page as shown in the screen shot below. I am happy to change how its implemented if you have suggestions. But this feature needs to exist on the item page. 


Example: 
image

Comment (Lemmings19) [reply resources/views/character/partials/header.blade.php:11]: Okay, I see where it's useful now. In that case, just remove the trailing `<li/>` please.

Comment (Pyrodius0) [reply resources/views/character/partials/header.blade.php]: Removed the <li/>. That was as artifact of a different implementation.

**Pull request merged**